mig/shared/functionality/cat.py

```diff
@@ -153,6 +173,12 @@ def main(client_id, user_arguments_dict, environ=None):
                                    'name': pattern})
             status = returnvalues.FILE_NOT_FOUND
 
+        if not _check_sizes(configuration, paths=match):
+            status = returnvalues.REJECTED_ERROR
```

Comment:
We have generally used `CLIENT_ERROR` for things not allowed, but it might make sense to have a specific return value for this. If so we just need to make sure it's actually handled all the way through the stack. Namely, that the output delivery helpers (html/txt/...-format as well as wsgi/cgi/...-handlers) know what to do about it.